ResearchKit.xcodeproj/project.pbxproj

```diff
@@ -1240,6 +1261,24 @@
 			path = ResearchKitTests;
 			sourceTree = "<group>";
 		};
+		A201A0C81BA1060200B7EEDC /* SurveyTasks */ = {
```

Comment:
I'm ambivalent towards this - I am happy to do so, but I had intended to match 'ActiveTasks' with 'SurveyTasks' so that the groups flowed (you might want to add some other sort of 'Task'.)